### ISSUE TITLE
Automatically show the answer but not the next question

### ISSUE THREAD

**Issue by ReneSac** [ankidroid/Anki-Android#4040]
Ankidroid has a feature to "Automatically display answer", that I can select, but then it forces me to also chose a "Time to show the next question" between 0 and 60 seconds, when I don't want that (for learning cards specially, I want to be able to read and think about it for much more than 1 minute). I tried setting to 0 seconds thinking it may be infinite, but it is indeed 0 seconds...

Is there any reason for this limitation?

**Comment by eginhard** [comment]
Duplicate of #3915 (you probably have gestures enabled as well if that happens...?)

**Comment by ReneSac** [comment]
No, I never used gestures, so I don't think it is really a duplicate. 

Even w/o using gestures it would mark failed and jump to the next card automatically... at least sometimes... I don't remember well... Maybe those "sometimes" was only when I finish editing a card (I just reproduced it now), and it was already annoying enough for me to stick to the 60 seconds until the bug #4039 happened and I stopped using this feature at all because it is bugged and unsafe (I couldn't erase the damage to the stats of my deck..).

With 60 seconds, even if the timer was already over 1 minute, it wouldn't jump to the next card when finishing a edition. Also, if you just go back from editing w/o changing anything even with 0 it don't jumps.

Anyway, my feature request is for a option to automatically show the answer, but not the next question. If it can be done using the number 0 of the slider that apparently is not doing what it is supposed to, ok. But maybe a separate option for that would be more intuitive.
